### PR TITLE
LPS-70828 Powered by doesn't translate into Chinese correctly

### PR DESCRIPTION
**Pull request by alee8888**
https://issues.liferay.com/browse/LPS-70828

@pei-jung @drewbrokke

### CODE REVIEW

File: modules/apps/frontend-theme/frontend-theme-classic/src/templates/portal_normal.ftl [hunk 1]
```diff
@@ -89,9 +89,12 @@
 				<div class="container">
 					<div class="row">
 						<div class="col-md-12 text-center text-md-left">
-							<@liferay.language key="powered-by" />
+							<#assign powered_by_liferay = '<a class="text-white" href="http://www.liferay.com" rel="external">Liferay</a>' />
 
-							<a class="text-white" href="http://www.liferay.com" rel="external">Liferay</a>
+							<@liferay.language_format
+								arguments="${powered_by_liferay}"
```
Comment (drewbrokke) [inline modules/apps/frontend-theme/frontend-theme-classic/src/templates/portal_normal.ftl:95]: Can this be inlined?

Comment (drewbrokke) [reply modules/apps/frontend-theme/frontend-theme-classic/src/templates/portal_normal.ftl:95]: It's an honest question, I have no idea whether or not Freemarker allows markup in assignment attributes.

**Review by drewbrokke — COMMENTED** [review]
@alee8888 this looks good. My only question has to do with whether or not we can inline those Freemarker variables. If not, then I think this is good to go. If so, let's see what it looks like when we do.

File: modules/apps/frontend-theme/frontend-theme-classic/src/templates/portal_normal.ftl (hunk 1 above)
Comment (alee8888) [reply modules/apps/frontend-theme/frontend-theme-classic/src/templates/portal_normal.ftl:95]: Just tested it and I can. Will resend, thanks!

Comment (drewbrokke) [reply modules/apps/frontend-theme/frontend-theme-classic/src/templates/portal_normal.ftl:95]: Sweet, go ahead and apply it to the other freemarker changes. Thanks Albert!

**Comment by alee8888** [comment]
Resent https://github.com/liferay-usm/liferay-portal/pull/121

**Pull request closed without merging**